ci/storage/test_main.sh

```diff
@@ -2,6 +2,26 @@
 
 set -eux
 
+cluster_reboot () {
+  # shellcheck disable=SC2029,SC2089
+  clush -B -S -o '-i ci_key' -l root -w "${tnodes}" reboot || true
+
+  # shellcheck disable=SC2029,SC2089
+  poll_cmd=( clush -B -S -o "-i ci_key" -l root -w "${tnodes}" )
+  poll_cmd+=( cat /etc/os-release )
+  reboot_timeout=600 # 10 minutes
+  retry_wait=10 # seconds
+  SECONDS=0
+  timeout=${SECONDS+$reboot_timeout}
```

Comment:
I'm not sure this is valid syntax.

```suggestion
  timeout=$((SECONDS + reboot_timeout))
```

Reply:
Will be fixed in the next push